Resources/Prototypes/_NF/Entities/Objects/Devices/flatpacks.yml

```diff
@@ -498,14 +498,29 @@
   - type: Flatpack
     entity: OreProcessorIndustrial
 
+# Mercenary
 - type: entity
-  parent: OreProcessorFlatpack
+  parent: BaseNFFlatpack
```

Comment:
Should this be done for others (e.g. SalvageTechfab?)  What was the benefit to the ore processor in particular?

Reply:
It is about inhering flatpack sprites and price tag, nothing else.